lib/ts/BufferWriter.h

```diff
@@ -259,10 +259,10 @@ class FixedBufferWriter : public BufferWriter
     const size_t newSize = _attempted + length;
 
     if (newSize <= _capacity) {
-      ink_assert(_buf != nullptr); // make clang-analyzer happy.
+      // ink_assert(_buf != nullptr); // make clang-analyzer happy.
```

Comment:
this is a temporary fix to make cache tool work.

Reply:
Although it breaks clang-analyzer.